include/picolibrary/microchip/mcp23008.h

```diff
@@ -23,10 +23,222 @@
 #ifndef PICOLIBRARY_MICROCHIP_MCP23008_H
 #define PICOLIBRARY_MICROCHIP_MCP23008_H
 
+#include <cstdint>
+
 /**
  * \brief Microchip MCP23008 facilities.
  */
 namespace picolibrary::Microchip::MCP23008 {
+
+/**
+ * \brief I/O Direction (IODIR) register.
```

Comment:
Move "register" into the register name and capitalize it for consistency with `picolibrary-microchip-megaavr` register definitions.